faq.md

```diff
@@ -132,6 +132,64 @@ Distributed writes to netCDF are hard.
 1. If you need to write to netCDF and your final dataset can fit in memory then use `dataset.load().to_netcdf(...)`.
 1. If you really must write a big dataset to netCDF try using `save_mfdataset` (see [here](https://ncar.github.io/xdev/posts/writing-multiple-netcdf-files-in-parallel-with-xarray-and-dask/)).
 
+### Fixing CESM monthly data time axis
+
+A well known issue of CESM data is that timestamps for fields saved as averages are placed at the end of the averaging period, and xarray does not treat this correctly when decoding the time axis. For instance, in the following example, the `January/1920` average has a timestamp of `February/1920`:
```

Comment:
xarray is treating it correctly! The file says the time is `midnight Feb 1 1920` which is the non-inclusive end of the time bound...

Reply:
> xarray is treating it correctly!  

You are right. I meant to mention that xarray isn't relying on the `bounds` information when decoding the `time` 

Removing this part shortly...